src/components/dar_collection_table/ResearcherActions.js

```diff
@@ -144,7 +114,7 @@ export default function ResearcherActions(props) {
   };
 
   const reviseButtonAttributes = {
-    keyProp: `revise-collection-${collectionId}`,
```

Comment:
why did you choose to change these from collectionId to uniqueId? Was the collectionId not unique?

Reply:
there isn't always a collection id. For example, if the dar collection is a draft,  it doesn't have a collection id, so it would be just 'undefined' here. Thus, we need to use a referenceId in the draft case.